DataManagementSystem/Service/FileCatalogHandler.py

```diff
@@ -524,16 +526,26 @@ def export_removeMetadata(self, pathMetadataDict):
   types_getDirectoryUserMetadata = [StringTypes]
 
   def export_getDirectoryUserMetadata(self, path):
-    """ Get all the metadata valid for the given directory path
+    """ Get all the metadata valid for the given directory path.
+        Client gets metadata w/o the VO suffix.
     """
-    return gFileCatalogDB.dmeta.getDirectoryMetadata(path, self.getRemoteCredentials())
+    metaDict = gFileCatalogDB.dmeta.getDirectoryMetadata(path, self.getRemoteCredentials(), enableStripping=False)
```

Comment:
This line (and corresponding debug output) has to be dropped, there is no point querying the database twice only for debug reasons